pkg/cmd/grafana-cli/commands/install_command_test.go

```diff
@@ -37,3 +39,39 @@ func TestFoldernameReplacement(t *testing.T) {
 		})
 	})
 }
+
+func TestExtractFiles(t *testing.T) {
+	Convey("Should preserve file permissions for plugin backend binaries for linux and darwin", t, func() {
+		err := os.RemoveAll("./testfiles/fake-plugins-dir/grafana-simple-json-datasource")
```

Comment:
rename the folder `testdata` instead. 

> This trick is used in the standard library. I learned it from Mitchell Hashimoto’s and Dave Cheney’s work. go test has good support for loading test data from files. Firstly, go build ignores directory named testdata. Secondly, when go test runs, it sets current directory as package directory. This allows you to use relative path testdata directory as a place to load and store your data. Here is an example:

https://medium.com/@povilasve/go-advanced-tips-tricks-a872503ac859